src/components/Jobs/RunningJobs/RunningJobs.js

```diff
@@ -1,28 +1,34 @@
-import React from 'react';
+import React, { useContext } from 'react';
+
+import { isEmpty } from 'lodash';
 
 import {
   sortRunningJobs,
   JobsListAccordion,
 } from '@folio/stripes-data-transfer-components';
 
-import { JOB_STATUSES } from '../../../utils/constants';
+import { JOB_EXECUTION_STATUSES } from '../../../utils/constants';
 import ItemFormatter from './ItemFormatter';
-import tempData from './tempData';
+import { DataFetcherContext } from '../../../contexts/DataFetcherContext';
+import { sampleJobExecution } from '../../../../test/bigtest/network/scenarios/fetch-job-profiles-success';
```

Comment:
As far as understand the data from tempData was moved fetch-job-profiles-success. Could you please clarify if tempData was deleted?

Reply:
I have firstly moved the file to the scenarios folder and then removed. Seems like github has identified that file was removed, but the `tempData` indeed is removed now.